pages/docs/contributing/maintainers.mdx

```diff
@@ -0,0 +1,87 @@
+import DocsLayout from "../../../components/layouts/DocsLayout.js";
+
+## Maintainers
+
+- Only assign issues with the label `status: ready for dev`
+- Each person should only be assigned 1 open issue at a time
+
+---
+
+> `avatar` json property is included
+
+This is no longer used, please leave an inline comment with the code sugestion to remove it - this will allow another maintainer to accept the changes and we can get the Pull Request merged without delay.
+
+```diff
+- "avatar": "http://github.com/SaraJaoude.png",
++
+```
+
+If you wish to make the fix yourself, you can merge to a temporary branch and fix, then create a new Pull Request.
+
+_Note: multiple Pull Requests that need fixing can go into this temporary branch, but please make sure this branch is not open for longer than a few hours to avoid any confusion._
```

Comment:
```suggestion
_Note: multiple Pull Requests that need fixing can go into a temporary branch, but please make sure it is not open for longer than a few hours to avoid any confusion._
```